python/DemoApp/core/utilities/plugins/mlb_basebase_data.py

```diff
@@ -0,0 +1,45 @@
+from typing import Annotated
+from datetime import datetime, timedelta
+from semantic_kernel import Kernel
+from semantic_kernel.functions import kernel_function
+#from semantic_kernel.functions.kernel_function_decorator import kernel_function
+from core.utilities.services.mlb_service import MlbService
+from core.utilities.formatters import ScheduleFormatter, TeamFormatter, GameFormatter, PlayByPlayFormatter
+
+class MlbBaseballData:
+    def __init__(self, mlb_service: MlbService):
+        self._mlb_service = mlb_service
+
+    @kernel_function(
+        description="Gets a list of games for a specific baseball team."
+    )
+    async def get_team_schedule_data(self, team_id: int) -> str:
```

Comment:
If you include ```from typing import Annotated```, then you can do something like ```team_id: Annotated[Date, "The id of the team being queried."]```. This helps a surprising amount during auto function invocation for consistent calling.

Reply:
You can also apply it to the output of a function. For example,
```async def get_team_ids_data(self) -> Annotated[str, "returns a dictionary of team ids and data pairs."]